generator/generator.go

```diff
@@ -421,12 +483,14 @@ func (g Generator) GenSchemaResolversFile() ([]byte, []*TypeDef) {
   g.P("")
   g.P("import (")
   g.In()
-  g.P(`"time"`)
+  // FIXME include only when time field is present
+  //g.P(`"time"`)
   g.P(`graphql "github.com/neelance/graphql-go"`)
```

Comment:
(Non blocking) Do we need another FIXME to make this link outside `generator.go`?